be/src/service/mem_hook.cpp

```diff
@@ -175,23 +178,22 @@ void operator delete[](void* p, size_t size, std::align_val_t al) noexcept {
 #define MEMORY_RELEASE_PTR(ptr) starrocks::tls_thread_status.mem_release(tc_malloc_size(ptr))
 #define MEMORY_CONSUME_SIZE(size) starrocks::tls_thread_status.mem_consume(size)
 #else
-#define TC_MALLOC_SIZE(ptr) 0
-#define MEMORY_CONSUME_PTR(ptr) (void)(ptr)
-#define MEMORY_RELEASE_PTR(ptr) (void)(ptr)
-#define MEMORY_CONSUME_SIZE(size) (void)(size)
+std::atomic<int64_t> g_mem_usage(0);
```

Comment:
could that varibale static and vebose name like 'global_memory_usage'?

Reply:
g_mem_usage will be used by test case directly

Reply:
The main purpose of using g_ is to be consistent in style with other code